packages/js/components/changelog/fix-select-control-popover-slots

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add name to exported popover slot so that it is not used for any popover items.
```

Comment:
"not used for any popover items" should probably be "not used for all popover items"

Reply:
Oh yeah... would probably be good to mention the context of what component we are talking about.

Reply:
🤣 yeah that is very ambiguous